changelogs/fragments/unset_environment_variables.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- Added ability to unset environment variables via shell plugins
```

Comment:
Include an explanation that this requires support from the shell plugin.

Reply:
Done